web/assets/js/main.js

```diff
@@ -35,9 +35,19 @@ $(function(){
     }
   };
 
+  var duplicateUsername = function(e) {
+    if (!disableCopying) {
```

Comment:
@sarahemm you'll want to have another variable `disableUsernameCopying` (also refactoring existing one to be `disableEmailCopying`) and define an `change` event handler for the username to disable the copying. See L52-L54 for email's example.